approval_polls/forms/choice_form.py

```diff
@@ -0,0 +1,12 @@
+__author__ = 'sparky'
+from django import forms
+from approval_polls.models import Choice
+
+
+class ChoiceForm(forms.ModelForm):
+    class Meta:
+        model = Choice
+        exclude = ('poll', )
+
+    choice_text = forms.CharField(label='Choice',
+                                  widget=forms.TextInput(attrs={'class': 'form-control'}))
```

Comment:
Good question because technically yes we can! BUT IMHO we shouldn't. Here is my rational.

The forms.Form and forms.ModelForm are part of Django's attempt to implement MVC. Conceptually the forms.Form or forms.ModelForm are related to the Controller part of the MVC model. And by convention has the is_valid method because this is where Django expects by design for form data to be validated. It has a number of hooks built into the "forms" module that do just that. 

If we push it aside and do the validation in the View we are violating the MVC as envisioned by Django. 

While in this instance the ChoiceForm is very simple it comes down to a separation of concerns and honouring that pattern in this case. But there is more ... 

Imagine a scenario where the validation for ChoiceForm has to very complex. Now you have code that rightly belongs in a controller class in your view class. Very messy. 

Also, it could be the case that we have to add extra validation for security reasons. What if we have malicious poll creator? What happens if escaped JS code is put into the ChoiceText? What if a malicious poll creator attempts an SQL injection attack via the ChoiceText field? etc. and so on.

Is the developer expected to check for all of that in the view? Would the developer know how? And why is the developer reimplementing validation for those sorts of things when they already exist in the Django forms framework and can easily access them by using forms.Form or forms.ModelForm?

So in my opinion, even though with the case of ChoiceForms being very simple and we could eschew typical Django forms in this case. I think it is a bad practice. 

That having been said. If you want it changed. We can change it. :)